hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -179,18 +179,21 @@ <h3 style="margin-top:40px"> NetCDF Header Info</h3>
     {% endif %}
 
 
-    {# Add netcdf opendap button  #}
-    {% if opendap_link %}
-        <td><a style="margin-left:15px" id="btn-netcdf-opendap" type="button" class="btn btn-success" href='{{opendap_link }}'
-               title="When changing this resource's sharing status, please allow 2 minutes for the OPeNDAP service to refresh on this page" >
-            <span class="button-label"> OPeNDAP Service</span>
-        </a></td>
+    {# Add THREDDS service #}
+    {% if nc_file_name %}
+        <div id="#thredds">
+            <h3>THREDDS Data Service</h3>
+            <h4>OPeNDAP: <a href="{{ opendap }}" target="_blank"> {{ nc_file_name }}</a></h4>
```

Comment:
it'd be good to add a space between {{ nd_file_name }} and </a> for consistency. Same applies to the following few statements as well, very minor formatting comment as a FYI.